k8s/sonarqube/chart/sonarqube/values.yaml

```diff
@@ -0,0 +1,24 @@
+replicaCount: 1
```

Comment:
This file looks diffrent than in other solutions. Please change it to fit one style

Reply:
it inst used so I removed it